### PR TITLE
feat(landingPage): serve landing page using server

### PR DESCRIPTION
**Pull request by armandopmj**

### CODE REVIEW

**Comment by armandopmj** [comment]
@mLuby: navigate to http://localhost:6060/ to be served landing page

File: server/config/express.middle.js [hunk 1]
```diff
@@ -1,9 +1,13 @@
 'user strict'
 
 var bodyParser = require('body-parser');
+var express = require('express');
+var dirnameArray =  __dirname.split('/');
+var path = dirnameArray.slice(0, dirnameArray.length - 1).join('/');
```
Comment (mLuby) [inline server/config/express.middle.js]: usually I see people using the node path module here: `var path = require('path')`
and then they do `path.join(__dirname, 'public')` down below

File: server/public/views.routing.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+var router = express.Router();
+
+router.get('/', function(req, res){
+    res.sendFile(__dirname + '/index.html');
+});
+router.get('/signup', function(req, res){
+    res.sendFile(__dirname + '/views/signup.html');
+});
+
+module.exports = router;
```
Comment (mLuby) [inline server/public/views.routing.js]: newline

**Comment by mLuby** [comment]
Nice! Made some comments, but none blocks merging this in.

File: server/config/express.middle.js (hunk 1 above)
Comment (armandopmj) [inline server/config/express.middle.js]: @mLuby shit that right... I'll update

File: server/public/views.routing.js (hunk 2 above)
Comment (armandopmj) [inline server/public/views.routing.js]: @mLuby jsLint is going to make catching this so much easier lol

**Comment by armandopmj** [comment]
@mLuby thanks for reviewing

**Comment by mLuby** [comment]
@armandopmj can I merge this in?

**Comment by armandopmj** [comment]
@mLuby go for it! I'm about to review the logging PR.

**Pull request merged**